Paccy10/olympus-api
# Feature(categories): create category

**What does this PR do?**

Adds create category functionality

**Description of Task to be completed?**

An authenticated admin user can create a new category

**How should this be manually tested?**

- Checkout to the branch `ft-create-category`
- Run the app with `flask run`
- Go to postman and send a `POST` request to /categories with the required payload
- A new category should be added to the database

**Any background context you want to provide?**

- N/A

**What are the relevant pivotal tracker stories?**

- N/A

**Screenshots (if appropriate)**

- N/A

**Questions:**

- N/A
